resources/event-bus/charts/event-publish-service/templates/_helpers.tpl

```diff
@@ -20,6 +20,7 @@ app.kubernetes.io/name: {{ template "event-publish-service.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/part-of: kyma
+app: {{ template "event-publish-service.name" . }}
```

Comment:
We had a PR some time ago to explicitly avoid those labels. Why are they back?

Reply:
These are not needed hence I removed it.